.github/workflows/build-android.yaml

```diff
@@ -34,6 +34,11 @@ on:
       SENTRY_DSN_ANDROID:
         required: true
 
+concurrency:
+  # non-deploy builds shouldn’t conflict, but builds for deployment can’t overlap or they’ll claim the same version number
+  group: build-android-${{ inputs.flavor }}${{ inputs.deploy && '' || github.run_id }}
```

Comment:
Oh, crap, this won’t work because `''` is falsy. (There is, I’m fairly certain, no other syntax for conditional expressions in GitHub Actions.)